doc/examples/tls.rst

```diff
@@ -118,24 +118,30 @@ PyMongo can be configured to present a client certificate using the
   ...                              tls=True,
   ...                              tlsCertificateKeyFile='/path/to/client.pem')
 
-If the private key for the client certificate is stored in a separate file use
-the ``ssl_keyfile`` option::
+If the private key for the client certificate is stored in a separate file,
+it should be concatenated with the certificate file. For example, to
+concatenate a PEM-formatted certificate file ``cert.pem`` and a PEM-formatted
+keyfile ``key.pem`` into a single file ``combined.pem``, on *nix systems users
```

Comment:
`*` means italics is RST. Does this render? Also everywhere else in the docs we just say "Unix" so maybe just change to that?

Reply:
Switched to Unix.